staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -162,22 +162,56 @@ $paths += "c:\AzureData\CustomDataSetupScript.log"
 Write-Host "Collecting Containerd running containers"
 $res = Get-Command ctr.exe -ErrorAction SilentlyContinue
 if ($res) {
-  Write-Host "Collecting Containerd running containers - containers"
+   Write-Host "Collecting ctr plugin ls"
+  & ctr.exe -n k8s.io plugin ls > "$ENV:TEMP\$timeStamp-containerd-plugin.txt"
+  $paths += "$ENV:TEMP\$timeStamp-containerd-plugin.txt"
+
+  Write-Host "Collecting ctr containers"
   & ctr.exe -n k8s.io c ls > "$ENV:TEMP\$timeStamp-containerd-containers.txt"
   $paths += "$ENV:TEMP\$timeStamp-containerd-containers.txt"
 
-  Write-Host "Collecting Containerd running containers - tasks"
+  Write-Host "Collecting ctr tasks"
   & ctr.exe -n k8s.io t ls > "$ENV:TEMP\$timeStamp-containerd-tasks.txt"
   $paths += "$ENV:TEMP\$timeStamp-containerd-tasks.txt"
 
-  Write-Host "Collecting Containerd running containers - snapshot"
+  Write-Host "Collecting ctr content ls"
+  & ctr.exe -n k8s.io content ls > "$ENV:TEMP\$timeStamp-containerd-content.txt"
+  $paths += "$ENV:TEMP\$timeStamp-containerd-content.txt"
+
+  Write-Host "Collecting ctr image ls"
+  & ctr.exe -n k8s.io image ls > "$ENV:TEMP\$timeStamp-containerd-image.txt"
+  $paths += "$ENV:TEMP\$timeStamp-containerd-image.txt"
+
+  Write-Host "Collecting ctr snapshot ls"
   & ctr.exe -n k8s.io snapshot ls > "$ENV:TEMP\$timeStamp-containerd-snapshot.txt"
   $paths += "$ENV:TEMP\$timeStamp-containerd-snapshot.txt"
+
+  Write-Host "Collecting ctr snapshot tree"
+  & ctr.exe -n k8s.io snapshot tree > "$ENV:TEMP\$timeStamp-containerd-snapshot-tree.txt"
+  $paths += "$ENV:TEMP\$timeStamp-containerd-snapshot-tree.txt"
+
+  Write-Host "Collecting ctr snapshot info for each snapshot"
+  $snapshotsList = & 'C:\Program Files\containerd\ctr.exe' -n k8s.io snapshot ls
+  foreach ($snapshot in $snapshotsList) {
+    $snapshotId = ($snapshot.Split(" ")[1])
+    $fileName = ($snapshotId.Split(":")[1])
+    & 'C:\Program Files\containerd\ctr.exe' -n k8s.io snapshot info $snapshotId > "containerd-snapshot-info-$fileName.txt"
+  }
 }
 else {
   Write-Host "ctr.exe command not available"
 }
 
+# collect disk usage information
+$logFolder = $pwd
+curl -Uri https://download.sysinternals.com/files/DU.zip -UseBasicParsing -OutFile DU.zip
+Expand-Archive .\DU.zip
+pushd DU
+  .\du.exe /accepteula
+  .\du.exe -l 1 C:\ProgramData\containerd\root\io.containerd.snapshotter.v1.windows\snapshots\ > $logFolder\du-snapshot-folder-size.txt
+popd
+Copy-Item 'C:\ProgramData\containerd\root\io.containerd.snapshotter.v1.windows\metadata.db' $logFolder\metadata.db
```

Comment:
@AbelHu  @ShiqianTao  is it ok to copy out the metadata db via the log script or refrain from doing this and ask cx to share it with us when needed?

Reply:
@ShiqianTao is this ok to do?

Reply:
Yes, thank you. Suggest using an absolute path, e.g., `c:\k\debug\du.exe`.